solution/generate_planning_solutions.go

```diff
@@ -116,6 +120,23 @@ func GenerateSolutions(context context.Context, timeMatcher *matching.TimeMatche
 	return
 }
 
+//TravelPlansDeduplication removes travel plans contain places that are permutations of each other
+func TravelPlansDeduplication(travelPlans []PlanningSolution) []PlanningSolution {
+	duplicatedPlans := make(map[string]bool)
+	results := make([]PlanningSolution, 0)
+
+	for _, travelPlan := range travelPlans {
+		placeIds := travelPlan.PlaceIDS
+		radix.Sort(placeIds)
```

Comment:
any specific reason to have radix sort? I believe it's O(n) in TC, but just wanted to understand why choose this ?

Reply:
Yes, for our case the N is small. But generally we want the linear time complexity.